run-checks

```diff
@@ -60,9 +60,27 @@ case "${1:-all}" in
     --spread-ubuntu)
         SPREAD=ubuntu-only
         ;;
+    --spread-ubuntu-lts)
+        SPREAD=ubuntu-lts
+        ;;
+    --spread-ubuntu-core)
+        SPREAD=ubuntu-core
+        ;;
+    --spread-ubuntu-next)
+        SPREAD=ubuntu-next
+        ;;
     --spread-no-ubuntu)
         SPREAD=no-ubuntu
         ;;
+    --spread-no-ubuntu-deb)
```

Comment:
I think it would be more natural to use `not-` rather than `no-`.

`--spread-not-ubuntu-deb`, `--spread-not-ubuntu-rpm` and `--spread-not-ubuntu-others`.